src/runtime/assemblymanager.cs

```diff
@@ -461,5 +458,13 @@ public static Type LookupType(string qname)
             }
             return null;
         }
+
+        internal static Type[] GetTypes(Assembly a)
```

Comment:
need a try-catch block here for transitive depedencies

Reply:
Isn't it an error, though, if we can't load transitive dependencies?